scanners/flutter/flutter.go

```diff
@@ -244,108 +159,129 @@ func (Scanner) ExcludedScannerNames() []string {
 func (scanner *Scanner) Options() (models.OptionNode, models.Warnings, models.Icons, error) {
 	flutterProjectLocationOption := models.NewOption(projectLocationInputTitle, projectLocationInputSummary, projectLocationInputEnvKey, models.TypeSelector)
 
-	for _, project := range scanner.projects {
-		var testKey string
-		if project.hasTest {
-			testKey = "test"
-		} else {
-			testKey = "notest"
-		}
-		cfg := configName + "-" + testKey
-
-		configOption := models.NewConfigOption(cfg+"-app-"+getBuildablePlatform(project.hasAndroidProject, project.hasIosProject), nil)
-		flutterProjectLocationOption.AddOption(project.path, configOption)
+	for _, proj := range scanner.projects {
+		configOption := models.NewConfigOption(proj.configName(), nil)
+		flutterProjectLocationOption.AddOption(proj.rootDir, configOption)
 	}
 
 	return *flutterProjectLocationOption, models.Warnings{}, nil, nil
 }
 
-func getBuildablePlatform(hasAndroidProject, hasIosProject bool) string {
-	switch {
-	case hasAndroidProject && !hasIosProject:
-		return "android"
-	case !hasAndroidProject && hasIosProject:
-		return "ios"
-	default:
-		return "both"
-	}
+var defaultProjects = []project{
+	{hasTest: true, hasAndroidProject: true, hasIosProject: true},
+	{hasTest: true, hasAndroidProject: false, hasIosProject: true},
+	{hasTest: true, hasAndroidProject: true, hasIosProject: false},
 }
 
 // DefaultOptions ...
-func (Scanner) DefaultOptions() models.OptionNode {
+func (scanner *Scanner) DefaultOptions() models.OptionNode {
 	flutterProjectLocationOption := models.NewOption(projectLocationInputTitle, projectLocationInputSummary, projectLocationInputEnvKey, models.TypeUserInput)
 
-	cfg := configName + "-test"
-
 	flutterPlatformOption := models.NewOption(platformInputTitle, platformInputSummary, "", models.TypeSelector)
-	flutterProjectLocationOption.AddOption("", flutterPlatformOption)
```

Comment:
empty string for user_input type could be a descriptive constant

Reply:
Updated in: https://github.com/bitrise-io/bitrise-init/pull/258/commits/78f2ffa6aa3ca8cc84b3323761c6ee621299e40f